pipeline.ipynb

```diff
@@ -152,7 +153,8 @@
     "            command=[\"python\", \"/code/predict.py\"],\n",
     "            pvolumes={\n",
     "                     \"/model/\": kfp.dsl.PipelineVolume(pvc=\"{{workflow.uid}}-model\"),\n",
-    "                     \"/code/\": kfp.dsl.PipelineVolume(pvc=\"{{workflow.uid}}-code\")\n",
+    "                     \"/code/\": kfp.dsl.PipelineVolume(pvc=\"{{workflow.uid}}-code\"),\n",
+    "                     \"/data/\": kfp.dsl.PipelineVolume(pvc=\"{{workflow.uid}}-featureset\")\n",
```

Comment:
is it data or featureset?

Reply:
It's the mount path for featureset, not the type.